src/Agent.Listener/Agent.Listener.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <OutputType>Exe</OutputType>
-    <RuntimeIdentifiers>win-x64;win-x86;linux-x64;linux-arm;osx-x64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win-x64;win-x86;linux-x64;linux-arm;rhel.6-x64;osx-x64</RuntimeIdentifiers>
```

Comment:
Note to reviewer: `rhel.6-x64` is the runtime identifier used by .NET Core. I'm also using the same identifier for the agent for consistency.